### PR TITLE
feat: #504 Add pre-configured container for a single node Elasticsearch

### PR DESCRIPTION
**Pull request by chertby**
Added configuration and test container for Elasticsearch.

### CODE REVIEW

File: testenvironments.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
```
Comment (HofmeisterAn) [inline testenvironments.json]: Is that file required?

Comment (chertby) [reply testenvironments.json]: My bad, thx

**Review by HofmeisterAn — APPROVED** [review]
LGTM, thanks for your contribution.

**Comment by HofmeisterAn** [comment]
https://github.com/testcontainers/testcontainers-dotnet/commit/1874b7f3ac8cd7793f751e9c66101a9cf5b60092, closes #504.
